common/src/stack/graph_ql/app/resolvers/NetworkResolver.py

```diff
@@ -4,11 +4,126 @@
 # https://github.com/Teradata/stacki/blob/master/LICENSE.txt
 # @copyright@
 
-
+import ipaddress
 from ariadne import ObjectType, QueryType, MutationType
 import app.db as db
 
 query = QueryType()
 mutation = MutationType()
 
-object_types = []
+@query.field("networks")
+def resolve_networks(_, info, id=None, name=None):
+    args = []
+    params = []
+    if id:
+        params.append(" AND id = %s ")
+        args.append(id)
+    if name:
+        params.append(" AND name = %s ")
+        args.append(name)
+    cmd = f"SELECT id, name, address, mask, gateway, mtu, zone, dns, pxe FROM subnets  WHERE 1 = 1 {''.join(params)}"
+    results, _ = db.run_sql(cmd, args)
+    return results
+
+@mutation.field("addNetwork")
+def resolve_add_network(_, info, name, address, mask, gateway=None, zone="", mtu=None, dns=False, pxe=False):
```

Comment:
I like that we decided on using `obj`, `info` instead of using the `_` when we plan on calling other resolveres.

Reply:
added obj.